hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -3743,6 +3743,15 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.snapshot.deep.cleaning.enabled</name>
+    <value>true</value>
```

Comment:
Why is the default value true for this patch  ?

Reply:
Should we have the default value true or false? THis is the question I had. We can have the value set to false by explicitly setting it.

Reply:
Let me set it to false by default.

Reply:
done